aea/cli/init.py

```diff
@@ -100,17 +96,11 @@ def do_init(
     config = get_or_create_cli_config()
     if reset or config.get(AUTHOR_KEY, None) is None:
         author = validate_author_name(author)
-        registry_type = validate_registry_type(registry_type)
```

Comment:
why would you remove validation? What if someone passes an invalid one?

Reply:
We don't pass registries as strings, we pass them using flags which have fixed values (`--remote | REGISTRY_REMOTE`, `--local | REGISTRY_LOCAL`, `--ipfs | REMOTE_IPFS` and `--http | REMOTE_HTTP`) so there's no way someone can provide wrong values